scripts/airflow/dags/copy_opendata_shapefiles.py

```diff
@@ -0,0 +1,44 @@
+"""
+copy_opendata_shapefiles
+
+Copy important datasets in SHP format over to the RDS.
+"""
+# pylint: disable=pointless-statement
+from datetime import datetime
+
+from airflow.operators.bash_operator import BashOperator
+
+from airflow_utils import create_dag
+
+START_DATE = datetime(2020, 2, 27)
+SCHEDULE_INTERVAL = '30 4 * * 6'
+DAG = create_dag(__file__, __doc__, START_DATE, SCHEDULE_INTERVAL)
+
+TASKS = {
```

Comment:
can we add a comment here explaining where these seemingly random UUIDs come from?